src/components/common.js

```diff
@@ -525,3 +526,19 @@ export const WarningTitle = ({ children }) => {
     children
   ])
 }
+
+export const ClipboardButton = ({ text, onClick, ...props }) => {
```

Comment:
TOL: do we want to put this in the other places that use it, like UriViewer etc? Or save that for another PR?